redis_broadcast.go

```diff
@@ -375,10 +373,12 @@ func (bc *redisBroadcast) Send(room, event string, args ...interface{}) {
 	defer bc.lock.RUnlock()
 
 	connections, ok := bc.rooms[room]
-	if ok {
-		for _, connection := range connections {
-			connection.Emit(event, args...)
-		}
+	if !ok {
+		return
```

Comment:
This changes the method. Previously `bc.publishMessage(room, event, args...)` was always called. Was that a bug or intentional?

Reply:
It is refactoring and not more

Reply:
This refactor changes the logic. If there is no room, it won't call publishMessage(), and other nodes won't receive the message.

Reply:
Sorry, guys. It is my fault. Erik, you right. Rollbacked